README.adoc

```diff
@@ -0,0 +1,106 @@
+= PartiQL Language Specification
+
+This is the link:AsciiDoc[https://asciidoc.org/] source for the link:https://partiql.org/[PartiQL] specification.
```

Comment:
Ah sorry, I mixed up the AsciiDoc arguments in the earlier suggestion. Should be:

```suggestion
This is the link:https://asciidoc.org/[AsciiDoc] source for the link:https://partiql.org/[PartiQL] specification.
```